SR2022/2022-04-02-competition-details-reminder.md

```diff
@@ -0,0 +1,40 @@
+---
+to: Student Robotics 2022
+subject: Competition details and important reminders
+---
+
+Hello all!
+
+It's just 3 weeks until the big day - [the competition](https://studentrobotics.org/events/sr2022/competition/)!
+
+Before then, we have a few important updates and reminders for you:
+
+## COVID Safety
+
+Life is beginning to get back to normal, and this means we are finally able to do in person events again! However, we must recognise that some people still have to be cautious, to protect their own health or their friends or family. As such, we ask that everyone follows [our COVID policy](https://studentrobotics.org/covid-19/) to help ensure we all keep safe, whilst still having lots of fun with robots!
+
+Please keep an eye on this as we get closer to the competition, as the content may change.
+
+## Robot design considerations
+
+At the competition, all robots undergo a safety check. This not only confirms that the robot satisfies our safety regulations, but also the rules. Of note are a particular few we often see teams forgetting until the last minute:
+
+- Make sure there are 2 spare USB ports: 1 for your code, and 1 for our "Competition USB". This Competition USB tells your robot that it's in the competition and which corner it's starting in (and thus populates the `R.zone` property). You will not be able to compete without this.
+- Make sure you have a way to mount the flag. We will have a few spare mounts, but please do not rely on this.
+- For safety, please make sure the power button is obvious and easily accessible. In the event of an issue in the arena, we don't want to be hunting for the switch as we try and quickly turn off your robot.
+- Ensure your battery is protected, ideally housed in its own rigid container
+- Make sure your robot fits inside the size constraints. If it doesn't, it will have a meeting with the _saw of shame_.
```

Comment:
It is worth thinking about the order here. The first thing should be the most important.

My feeling is they should be ordered according to:
1. safety
2. rules compliance
3. functional

A robot without an accessible power button or an exposed battery is a danger everywhere, be it the arena, the test arena, or the pits.
A robot that's too large will disqualify teams from matches and be inconvenient.
A robot without enough USB ports is ultimately safe, and is rules compliant, just not very useful.